src/Razor/src/Microsoft.NET.Sdk.Razor/build/netstandard2.0/Microsoft.NET.Sdk.Razor.CodeGeneration.targets

```diff
@@ -156,6 +156,7 @@ Copyright (c) .NET Foundation. All rights reserved.
       PipeName="$(_RazorBuildServerPipeName)"
       Version="$(RazorLangVersion)"
       RootNamespace="$(RootNamespace)"
+      CSharpLanguageVersion="$(LangVersion)"
```

Comment:
I think we should also double-check https://github.com/aspnet/AspNetCore-Tooling/blob/master/src/Razor/src/Microsoft.NET.Sdk.Razor/build/netstandard2.0/Microsoft.NET.Sdk.Razor.Compilation.targets#L107 and see if anything needs to change or be added.